chesster.js

```diff
@@ -161,6 +152,17 @@ function update_channels(bot){
     });
 
 }
+function refreshCurrentRoundSchedules() {
+    var _45_45 = league.getLeague("45+45", config);
+    _45_45.getCurrentRoundSchedule(function(err, pairings) {
```

Comment:
I agree. Although I think that the league will want to control how often each gets called.  For instance, getRosters only needs to be called once per week (ish) while getCurrentRoundSchedule will need to be called far more often, like it is here.

Those times could come from the config and perhaps it's more of a:

```_45_45.installRefreshTimeouts()``` or something?